### ISSUE TITLE
Cannot edit slider dots styles because CSS @apply has been removed

### ISSUE THREAD

**Issue by trekkete** [FlowingCode/CarouselAddon#41]
In your code you use properties:

* `--paper-slider-dot-container-styles` | (Mixin) Custom styles for dot container | NULL
* `--paper-slide-dot-styles` | (Mixin) Custon styles for dots | NULL

that you then `@apply` to the styles of the relative components.

This feature has been removed from browsers, so there is no way to edit the slider dots appearance. 

Could you change it in some other way?

Thank you

**Comment by javier-godoy** [comment]
Hello. We'll investigate this issue. In version 2.1.4 (that we released today) we forked the web-component, so now we could refactor it in order to support [shadow DOM styling](https://vaadin.com/docs/latest/styling/advanced/shadow-dom-styling) and/or stylable parts.

**Comment by javier-godoy** [comment]
Hello. We've just released version 3.0.0.

It's a breaking change because the web-component is now `fc-l2t-paper-slider` instead of `l2t-paper-slider`, but that's the only change. The Java API is compatible with version 2.x of the add-on.

The new version leverages Vaadin shadow DOM styling mechanism (see demo [here](https://addonsv23.flowingcode.com/carousel/custom-theme)), so that you can annotate the view with `@CssImport(value = "./carousel-demo-styles.css", themeFor = "fc-l2t-paper-slider")` and add some CSS in `frontend/carousel-demo-styles.css`

```
:host([theme~="custom-theme"]) .slider__dots > span { 
	background: purple;
}
```
